### ISSUE TITLE
Review DCR Instance Types

### ISSUE THREAD

**Issue by ioannakok** [guardian/dotcom-rendering#9321]
We're currently using `t4g.small` in production:

https://github.com/guardian/dotcom-rendering/blob/e234547e28cb1071b83c526fdbae7df5c361f522/dotcom-rendering/scripts/deploy/riff-raff.yaml#L16-L19

Do we want to consider other instance types? Frontend uses a different type, as mentioned in #7440.

#### Things to consider
* What about the reservations we've already made for the year (they happen in March)? We can evaluate now and make the new reservations with what we decide
* If we go for a larger instance, this will unblock the work @arelra has started on turning on cluster mode in DCR instances. See more on why this would be beneficial and the work already done here:
  - https://github.com/guardian/dotcom-rendering/pull/8789
  - https://github.com/guardian/dotcom-rendering/pull/8929
  - https://github.com/guardian/dotcom-rendering/pull/9105

#### Useful docs
* https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/apply_ri.html#ri-normalization-factor

**Comment by alinaboghiu** [comment]
https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/docs/architecture/029-scaling-policy-decision-record.md
